### PR TITLE
SUPESC-600: Modified docs and diagram due to the changed functionality

### PR DESCRIPTION
**Pull request by Zingeon**
## PR Description

Ticket: https://spryker.atlassian.net/browse/SUPESC-600

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

**Comment by ilyakubanov** [comment]
Why are you changing 202204.0 release docs? These code doesn't exist in 202204.0, must be presented for the next release

File: docs/marketplace/dev/glue-api-guides/202204.0/guest-carts/managing-guest-carts.md [hunk 1]
```diff
@@ -1944,9 +1944,24 @@ For the attributes of other included resources, see:
 
 ## Assign a guest cart to a registered customer
 
-Guest carts are anonymous as they are not related to any user. If a user registers or logs in, the guest cart is automatically assigned to their account.
+Guest carts are anonymous as they are not related to any user. If a user registers or logs in, the guest cart can be automatically assigned to their account.
 
-To assign a guest cart to a customer, that is, merge the carts, include the unique identifier associated with the customer in the *X-Anonymous-Customer-Unique-Id* header of the authentication request if it is an existing customer, or request to create a customer account if it is a new one.
+To assign a guest cart to a customer, that is, merge the carts, include the unique identifier associated with the customer in the `X-Anonymous-Customer-Unique-Id` header of the authentication request if it is an existing customer, or request to create a customer account if it is a new one. Also, you need to adjust the configuration constant:
```
[the remark this answers is not in the document]
Comment (m-alama) [reply docs/marketplace/dev/glue-api-guides/202204.0/guest-carts/managing-guest-carts.md:1949]: the reason for introducing this config constant based on this conversation 
https://github.com/spryker/spryker/pull/9446#discussion_r903582122

**Comment by dmytro-sokolovskyi** [comment]
@ilyakubanov It's a good point. Thank you.
@Zingeon We should create a separate folder named `upcoming-release` instead of using `202204.0` and copy related files with updates there. In the current files, you need to roll back all changes.

File: docs/marketplace/dev/glue-api-guides/202204.0/guest-carts/managing-guest-carts.md [hunk 2]
```diff
@@ -1955,9 +1970,10 @@ Upon login, the behavior depends on whether your project is a single cart or [m
 
 The workflow is displayed in the following diagram:
 
-![Assign cart](https://spryker.s3.eu-central-1.amazonaws.com/docs/Glue+API/Glue+API+Storefront+Guides/Managing+Carts/Managing+Guest+Carts/assigning-guest-cart-to-registered-user.png)
```
[the remark this answers is not in the document]
Comment (Zingeon) [reply docs/marketplace/dev/glue-api-guides/202204.0/guest-carts/managing-guest-carts.md:1958]: The diagram is embed HTML from  https://app.diagrams.net/ and it compiles fine after installing the application. You can check it [here](https://drive.google.com/file/d/17KE_KQjCrvFhszSxXn_XVpUp6BZnCMeu/view)

File: docs/scos/dev/glue-api-guides/upcoming-release/managing-carts/guest-carts/managing-guest-carts.md [hunk 3]
```diff
@@ -0,0 +1,1239 @@
...
+<script type="text/javascript" src="https://viewer.diagrams.net/js/viewer-static.min.js"></script>
+
+Below, you can see an exemplary workflow for converting a guest cart into a regular cart:
```
Comment (dmytro-sokolovskyi) [inline docs/scos/dev/glue-api-guides/upcoming-release/managing-carts/guest-carts/managing-guest-carts.md:1074]: Make sense to fix it like in the marketplace doc:
`The following is an exemplary workflow of converting a guest cart into a regular cart:`

File: docs/scos/dev/glue-api-guides/upcoming-release/managing-carts/guest-carts/managing-guest-carts.md [hunk 4]
```diff
@@ -0,0 +1,1239 @@
...
+* [Gift Cards of Guest Users](/docs/scos/dev/glue-api-guides/{{page.version}}/managing-carts/guest-carts/managing-gift-cards-of-guest-users.html)
+* [Retrieve a measurement unit](/docs/scos/dev/glue-api-guides/{{page.version}}/retrieving-measurement-units.html#measurement-units-response-attributes)
+* [Retrieving product labels](/docs/scos/dev/glue-api-guides/{{page.version}}/managing-products/retrieving-product-labels.html#product-labels-response-attributes)
+
+## Assign a guest cart to a registered customer
+
+Guest carts are anonymous as they are not related to any user. If a user registers or logs in, the guest cart can be automatically assigned to their account.
+
+To assign a guest cart to a customer, for example, merge the carts, include the unique identifier associated with the customer in the *X-Anonymous-Customer-Unique-Id* header of the authentication request if it is an existing customer, or request to create a customer account if it is a new one. Adjust the configuration constant to create a cart for the newly authenticated customer while merging the guest cart with the customer cart:
```
Comment (dmytro-sokolovskyi) [inline docs/scos/dev/glue-api-guides/upcoming-release/managing-carts/guest-carts/managing-guest-carts.md:1048]: `X-Anonymous-Customer-Unique-Id`

File: docs/scos/dev/glue-api-guides/upcoming-release/managing-carts/guest-carts/managing-guest-carts.md [hunk 5]
```diff
@@ -0,0 +1,1239 @@
...
+            "id": "9183f604-9b2c-53d9-acbf-cf59b9b2ff9f",
+            "attributes": {...},
+            "links": {...}
+        }
+    ],
+    "links": {...}
+}
+```
+
+In a **single cart** environment, items from the guest cart have been added to the user's own cart.
```
Comment (dmytro-sokolovskyi) [inline docs/scos/dev/glue-api-guides/upcoming-release/managing-carts/guest-carts/managing-guest-carts.md:1188]: `*single cart*`

**Review by dmytro-sokolovskyi — DISMISSED** [review]

**Review by VadymSachenko — APPROVED** [review]

**Pull request merged**